src/vs/platform/environment/node/argv.ts

```diff
@@ -127,6 +129,7 @@ export function parseArgs(args: string[]): ParsedArgs {
 }
 
 export const optionsHelp: { [name: string]: string; } = {
+	'-': isWindows ? localize('stdinWindows', "Read output from another program (e.g. 'echo Hello World | {0} -').", product.applicationName) : localize('stdinUnix', "Read from stdin (e.g. 'ps aux | grep code | {0} -').", product.applicationName),
```

Comment:
`-` isn't an option, I wouldn't put it here since options can be put in any position of the CLI. `-` seems to be only placeable where files are. I would either omit this advanced option in the help message, or simply document it further down: _instead of a file path you can also use `-` to read input from stdin_. Something like that.

Reply:
@joaomoreno good point, please check https://github.com/Microsoft/vscode/commit/dfbff7fa99768a17734a65396942d25358a7e5f3